docs/_templates/class.rst

```diff
@@ -31,6 +37,10 @@
 
 .. |filename| replace:: ``filename`` (:obj:`str`): the filename of video/image being read.
 
+.. |fps| replace:: ``fps`` (:obj:`list`): a numpy array (N) of the FPS per frame. The 
```

Comment:
is this a list or numpy array? or list of numpy arrays?

Reply:
a list of floats, I have updated accordingly